check_for_update.go

```diff
@@ -50,12 +50,13 @@ func getLatestVersion() (*semver.Version, error) {
 	if err != nil || time.Since(modified) > 24*time.Hour {
 		versionOnline, err := getLatestVersionFromURL()
 		if err == nil {
-			version = versionOnline
-			ioutil.WriteFile(versionCacheFilename, []byte(version.String()), 0600)
+			ioutil.WriteFile(versionCacheFilename, []byte(versionOnline.String()), 0600)
 		}
+
+		return versionOnline, err
 	}
 
-	return version, nil
+	return version, err
```

Comment:
Is this related to https://github.com/phrase/phraseapp-client/issues/79 ?

Is it possible that we reach this point and where err is not nil? If error is is expected to always be nil we should also always return an explicit nil so that we do not pick up an old error we are not interested in (like if we decide to handle the error on writing the cache file in the future).